compiler/riscv/codegen/FPTreeEvaluator.cpp

```diff
@@ -605,13 +714,38 @@ commonFpMinMaxEvaluator(TR::Node *node, TR::InstOpCode::Mnemonic op, bool revers
    deps->addPostCondition(trgReg, TR::RealRegister::NoReg);
    deps->addPostCondition(src2Reg, TR::RealRegister::NoReg);
 
+   generateLABEL(cg, TR::InstOpCode::label, node, startLabel);
```

Comment:
Now that this label has been moved, `src1Reg` is also used during ICF. Because of that, it needs to be added to the list of dependencies.